site/content/entry/yuzu-progress-report-oct-2023/index.md

```diff
@@ -0,0 +1,412 @@
+Images were being {{< gh-hovercard "11744" "marked as rescaled" >}}, even if the resolution scaler was not in use (running at 1x). 
+This led to a slight overhead and some asserts.
+While no game bug was known to be caused by this, it’s good to have preemptive fixes for once instead of just reactionary ones.
+
+In the meantime, Maide has also been removing image alias bits for all image attachments in an effort to allow the drivers to use more memory optimizations.
+This pull request also includes {{< gh-hovercard "11747" "some other minor fixes" >}} with it.
+
+By {{< gh-hovercard "11775" "implementing" >}} the first and subsequent draw commands for vertex arrays, Super Meat Boy finally renders correctly! No more black screens!
+
+{{< imgs
+	"./smb.png| Well done (Super Meat Boy)"
+  >}}
+
+And now, one for the Linux gang.
+[v1993](https://github.com/v1993) tested and {{< gh-hovercard "11786" "re-enabled CUDA video decoding" >}} on Linux, allowing better video decoding performance for NVIDIA users (running the proprietary driver, of course).
+We previously disabled CUDA by default because it can fail on systems running both a dedicated NVIDIA GPU and a dedicated AMD GPU (iGPUs are fine), a decently rare configuration that only a few people, like your writer, actually ever use.
+
+For those few users running mixed hardware vendors on your systems, please manually  select “CPU Video Decoding” if you’re affected by video decoding issues now. 
+This was the previous default behaviour.
+
+We got user reports of crashes happening when grabbing a Grand Star in `Super Mario Galaxy`, as part of `Super Mario 3D All-Stars`.
+byte[] found that the problem is in how the Vulkan scheduler incorrectly flushes data.
+The solution? [Use a lock](https://www.youtube.com/watch?v=SNgNBsCI4EA). 
+And if that doesn’t work, {{< gh-hovercard "11806" "use more locks." >}}
+Happy Star hunting!
+
+To improve OpenGL support even further beyond, [Epicboy](https://github.com/ameerj) returns.
+First, he found that the `shfl_in_bounds` function, which is used to perform a warp-level parallel reduction in compute, could cause some threads to be inactive and return invalid results. 
+{{< gh-hovercard "11847" "The solution" >}} was to move the `shfl_in_bounds` check after the `readInvocationARB` function, which requires all threads to be active, to avoid ~~dragons~~ undefined behaviour. 
+This fixed some graphical corruption issues in unit tests, which should lead to fixes in real games too.
+
+Next, a simple gift from epicboy: {{< gh-hovercard "11904" "force enabling" >}} `Threaded optimization`, an NVIDIA specific OpenGL optimization that enables the use of a new separate CPU thread for graphics rendering.
+This is a solid performance boost for those running OpenGL with NVIDIA hardware.
+And for those asking, yes, Vulkan assigns a separate thread for rendering too–it can be done on an API level, instead of with a driver-level setting.
```

Comment:
```suggestion
And for those asking, yes, Vulkan allows the use of a separate thread for rendering too–but we do it on an API level, instead of with a driver setting, so all GPUs can benefit.
```